doc/tut1.rst

```diff
@@ -595,8 +595,10 @@ Procedures
 
 To define new commands like `echo <system.html#echo,varargs[typed,]>`_
 and `readLine <io.html#readLine,File>`_ in the examples, the concept of a
-*procedure* is needed. (Some languages call them *methods* or *functions*.)
-In Nim new procedures are defined with the `proc` keyword:
+*procedure* is needed. You might be used to them being called *methods* or
+*functions* in other languages, but Nim
```

Comment:
This and the next line have Nim

Reply:
Didn't consider that paragraph as reworked :)

Reply:
I was playing by smelt it dealt it rules. 😁

OK, it can live this PR, but its day will come.